src/test/java/io/anserini/doc/GenerateReproductionDocsTest.java

```diff
@@ -41,12 +41,16 @@ public class GenerateReproductionDocsTest {
   public final static String COLLECTION = "msmarco-v1-passage";
   public final static String[] MODELS = {
       "bm25-default",
-      "splade-pp-ed-cached_q",
-      "splade-pp-ed-onnx",
-      "cos-dpr-distil-cached_q",
-      "cos-dpr-distil-onnx",
-      "bge-base-en-15-cached_q",
-      "bge-base-en-15-onnx"
+      "splade-pp-ed.cached_q",
+      "splade-pp-ed.onnx",
+      "cos-dpr-distil.fp32.cached_q",
+      "cos-dpr-distil.fp32.onnx",
+      "cos-dpr-distil.int8.cached_q",
+      "cos-dpr-distil.int8.onnx",
+      "bge-base-en-15.fp32.cached_q",
+      "bge-base-en-15.fp32.onnx",
+      "bge-base-en-15.int8.cached_q",
+      "bge-base-en-15.int8.onnx"
   };
 
   public static String findMsMarcoTableTopicSetKeyV1(String topicKey) {
```

Comment:
looks like some of the newly added topic keys have trec2020 or treck2019 instead of dl20 or dl19, should they be covered in the tests too?

Reply:
Actually, I already have test cases for aliases: https://github.com/castorini/anserini/blob/master/src/test/java/io/anserini/search/topicreader/TopicsTest.java#L40

Will add more.